docs/user-documentation/loading-media.md

```diff
@@ -0,0 +1,34 @@
+To load media in Islandora, first click on Add content. This is under Tools.
+![Click on add content](/assets/Add content.jpg)
```

Comment:
`/assets/Add content.jpg` → `../assets/Add content.jpg` and rename the jpegs to be lower-case and use underscores or hyphens instead of spaces. (Same with the others.) Progress! 👍